src/components/DropzoneAreaBase.js

```diff
@@ -35,7 +35,7 @@ const styles = ({palette, shape, spacing}) => ({
         cursor: 'pointer',
         overflow: 'hidden',
         flexDirection: 'column',
-        justify: 'center',
+        justifyContent: 'center',
```

Comment:
Yes that's right actually! it needs to be like that for jss, but the other way round for the grid Props